### PR TITLE
engine: hack to allow us to watch all containers of a pod for logs [ch1737]

### PR DESCRIPTION
**Pull request by maiamcc**
This is the quickest way to support multi-container logs.

It doesn't actually improve our model in any useful way -- that's
for a separate PR.

### CODE REVIEW

**Comment by maiamcc** [comment]
It looks real ugly, but it does the thing:

screen shot 2019-03-05 at 3 43 19 pm
screen shot 2019-03-05 at 3 43 27 pm

**Comment by landism** [comment]
I'd expect this to show synclet logs, which we probably don't want. Have you tested this with a fast build in GKE?

File: internal/engine/podlogmanager.go [hunk 1]
```diff
@@ -50,9 +56,18 @@ func (m *PodLogManager) diff(ctx context.Context, st store.RStore) (setup []PodL
 				continue
 			}
 
-			watchByPod, ok := m.watchesByPod[pod.PodID]
-			if ok && watchByPod.cID != pod.ContainerID {
-				watchByPod.cancel()
+			// ASDFG what?!
+			// maybe check for main container id and shut down all if main cID doesn't match?
```
Comment (landism) [inline internal/engine/podlogmanager.go]: this comment block feels more like a WIP comment than one we'd want to have to interpret in a few weeks

Comment (maiamcc) [reply internal/engine/podlogmanager.go]: lol indeed, this is a WIP comment 😅 deleted

**Comment by maiamcc** [comment]
> I'd expect this to show synclet logs, which we probably don't want. Have you tested this with a fast build in GKE?

yep, was just talking to DM about this. fixed!

**Pull request merged**